CONTRIBUTING.md

```diff
@@ -1,4 +1,7 @@
-# Contributing
+# Contents
```

Comment:
Please make sure Jim reviews this before checking in.

Reply:
Summoning @jperrin

Reply:
Done :)